dev-utils/karma.js

```diff
@@ -35,7 +35,7 @@ const baseLaunchers = getBrowserList().map(launcher => ({
   ...launcher
 }))
 const specPattern =
-  'test/{*.spec.+(js|ts),!(e2e|integration|node|bundle)/*.spec.+(js|ts)}'
+  'test/{*.spec.+(js|ts),!(e2e|integration|node|bundle|types)/*.spec.+(js|ts)}'
```

Comment:
Why the `types` is necessary here?

Reply:
To ignore from running in karma as part of unit tests.